flip-shop_app/src/pages/products/Product.jsx

```diff
@@ -1,55 +1,49 @@
 import "./product-filter.css";
-// import { products } from "../../data";
-import { productImg1, productImg2 } from "../../assets/images/photos";
+import axios from 'axios';
+import React, { useEffect, useState } from "react";
 import { ProductsList } from "./components/product-all";
 import { Productfilter } from "./components/product-filter";
+
 export default function ProductWithFilter() {
+  const [productData, setProductData] = useState([]);
+  useEffect(() => {
+    (async function () {
+      try {
+        await axios.get("/api/products").then((res) => {
```

Comment:
The whole point of using async-await is to avoid using `.then` you should write it as:- 
`const response = await axios.get("/api/products");`
And then check `response.status` if it is `200` then only set the products otherwise don't.